pkg/kubectl/resource_printer.go

```diff
@@ -352,23 +377,48 @@ func NewHumanReadablePrinter(noHeaders, withNamespace bool, wide bool, showAll b
 			AbsoluteTimestamps: absoluteTimestamps,
 			ColumnLabels:       columnLabels,
 		},
+		hiddenPodsNum: 0,
 	}
 	printer.addDefaultHandlers()
 	return printer
 }
 
 // Handler adds a print handler with a given set of columns to HumanReadablePrinter instance.
 // See validatePrintHandlerFunc for required method signature.
-func (h *HumanReadablePrinter) Handler(columns []string, printFunc interface{}) error {
+func (h *HumanReadablePrinter) Handler(columns []string, printFuncsArgs ...interface{}) error {
```

Comment:
This has serious code smell. Instead of hacking the handler interface to pass arbitrary args, suggest using a wrapper function around `printPod`, `printPodList` as the default handler. See comment in `addDefaultHandlers()`.